docs/how_to/pytorch_install/pytorch_install.md

```diff
@@ -1,377 +1,399 @@
 
    ```bash
-   ./.jenkins/pytorch/build.sh
+   ./.ci/pytorch/build.sh
    ```
 
-   This will first convert PyTorch sources for HIP compatibility and build the
+   This converts PyTorch sources for
+   [HIP compatibility](https://www.amd.com/en/developer/rocm-hub/hip-sdk.html) and builds the
    PyTorch framework.
 
-8. Alternatively, build PyTorch by issuing the following commands:
+   To check if your build is successful, run:
 
    ```bash
-   python3 tools/amd_build/build_amd.py
-   USE_ROCM=1 MAX_JOBS=4 python3 setup.py install --user
+   echo $? # should return 0 if success
    ```
 
-#### Option 4: Install Using PyTorch Upstream Docker File
-
-Instead of using a prebuilt base Docker image, you can build a custom base
-Docker image using scripts from the PyTorch repository. This will utilize a
-standard Docker image from operating system maintainers and install all the
-dependencies required to build PyTorch, including
+## Using the PyTorch upstream Docker file
 
-- ROCm
-- Torchvision
-- Conda packages
-- Compiler toolchain
+If you don't want to use a prebuilt base Docker image, you can build a custom base Docker image
+using scripts from the PyTorch repository. This uses a standard Docker image from operating system
+maintainers and installs all the required dependencies, including:
 
-Follow these steps:
+* ROCm
+* Torchvision
+* Conda packages
+* The compiler toolchain
 
-1. Clone the PyTorch repository on the host.
+1. Clone the PyTorch repository.
 
    ```bash
    cd ~
    git clone https://github.com/pytorch/pytorch.git
-   cd pytorch
+   cd /pytorch
    git submodule update --init --recursive
    ```
 
 2. Build the PyTorch Docker image.
 
    ```bash
-   cd.circleci/docker
-   ./build.sh pytorch-linux-bionic-rocm<version>-py3.7
-   # eg. ./build.sh pytorch-linux-bionic-rocm3.10-py3.7
+   cd .ci/docker
+   ./build.sh pytorch-linux-<os-version>-rocm<rocm-version>-py<python-version> -t rocm/pytorch:build_from_dockerfile
    ```
 
-   This should be complete with a message "Successfully build `<image_id>`."
+   Where:
+   * `<os-version>`: `ubuntu20.04` (or `focal`), `ubuntu22.04` (or `jammy`), `centos7.5`, or `centos9`
+   * `<rocm-version>`: `5.4`, `5.5`, or `5.6`
+   * `<python-version>`: `3.8`-`3.11`
 
-3. Start a Docker container using the image:
+   To verify that your image was successfully created, run:
+
+   `docker image ls rocm/pytorch:build_from_dockerfile`
+
+   If successful, the output looks like this:
 
    ```bash
-   docker run -it --cap-add=SYS_PTRACE --security-opt
-   seccomp=unconfined --device=/dev/kfd --device=/dev/dri --group-add
-   video --ipc=host --shm-size 8G <image_id>
+   REPOSITORY    TAG                       IMAGE ID         CREATED           SIZE
+   rocm/pytorch  build_from_dockerfile     17071499be47     2 minutes ago     32.8GB
    ```
 
-   You can also pass -v argument to mount any data directories from the host
-   onto the container.
+3. Start a Docker container using the image.
```

Comment:
@LisaDelaney 3. Start a Docker container using the image **with mounted Pytorch folder**:
Here we notice that we are pass the Pytorch folder downloaded in the previous steps into a container.